kiali-server/values.yaml

```diff
@@ -18,6 +18,11 @@ kiali_route_url: ""
 # Note that only those values used by the Helm Chart will be here.
 #
 
+additional_display_details:
+- annotation: kiali.io/api-spec
+  icon_annotation: kiali.io/api-type
+  title: API Documentation
+
```

Comment:
Is this really needed? Note the comment above that says, `only those values used by the Helm Chart will be here`. We only put stuff in here that the actual Helm charts reference and use. We don't set simple defaults here as there should be no need.

Reply:
Hmm... I guess this isn't a hardcoded default in the server? I'm trying to think if we really want this in the values file or not - we never put defaults in here unless the charts actually use them, but in this case I don't see a way around having to put this in here.

Reply:
OK, I think the difference here is that this setting is an array of items. So if we hardcode this default, but the user sets his own, that default list item will be overwritten. The user will need to know to add it back in if they want to see the Kiali API stuff.  So, yes, I can see why we could put this in there, because it expicitly shows the user the kiali API spec list item and if they want their own, they can just add it to the list if they want to keep the kiali one.

Reply:
right, this is not hardcoded values but says where to look the values